generators/Generators.fs

```diff
@@ -100,6 +100,21 @@ type CryptoSquare() =
 type DifferenceOfSquares() =
     inherit Exercise()
 
+type Dominoes() =
+    inherit Exercise()
+    
+    let formatAsTuple (value:obj) =
+        let twoElementList = value :?> JArray |> normalizeJArray
+        (twoElementList.Item 0, twoElementList.Item 1) |> string
+
+    override this.RenderInput (canonicalDataCase, key, value) =
+        value :?> JArray
+        |> normalizeJArray
+        |> Seq.map formatAsTuple
+        |> formatList
+
+    override this.PropertiesWithIdentifier canonicalDataCase = ["input"]
```

Comment:
With #415 merged, you can now also use `this.Properties canonicalDataCase` (see [example](https://github.com/exercism/fsharp/pull/415/files#diff-378e45c2fa29f82b16b9d312d5f5c18aR173)) instead of being explicit when defining the properties with an identifier. The benefit of that is that renamed values don't break the generator, although it also means that new properties will automatically have an identifier. In this case I'm partial to either solution, what do you think?

Reply:
I agree with the idea of using `this.Properties canonicalDataCase` rather than defining the properties with an identifier. Will change to use  `this.Properties canonicalDataCase`.
Thanks @ErikSchierboom

Reply:
@vrnithinkumar Ah, I linked to the wrong property, as `this.Properties` also includes the expected value, which doesn't need to be included. Could you use `this.PropertiesUsedAsSutParameter` instead? Sorry about that.

Reply:
Made changes to use  Using PropertiesUsedAsSutParameter not Properties